installers-conda/build_installers.py

```diff
@@ -5,523 +5,331 @@
         background.paste(_logo.resize((148, 148)), (95, 418))
-        output = WELCOME_IMG_MAC
+        output = BUILD / "welcome_img_mac.png"
         background.save(output, format="png")
 
-
-def _get_condarc():
-    contents = dedent(
-        """
-        channels:  #!final
-          - conda-forge/label/spyder_rc
-          - conda-forge/label/spyder_dev
-          - conda-forge/label/spyder_kernels_rc
-          # TODO: Add this channel when we create it
-          # - conda-forge/label/spyder_kernels_dev
-          - conda-forge
-        repodata_fns:  #!final
-          - repodata.json
-        auto_update_conda: false  #!final
-        notify_outdated_conda: false  #!final
-        channel_priority: flexible  #!final
-        env_prompt: '[spyder]({default_env}) '  #! final
-        register_envs: false  #! final
-        """
-    )
-    # the undocumented #!final comment is explained here
-    # https://www.anaconda.com/blog/conda-configuration-engine-power-users
-    file = BUILD / "condarc"
-    file.write_text(contents)
-
-    return str(file)
-
-
-def _get_conda_bld_path_url():
-    bld_path_url = "file://"
-    if WINDOWS:
-        bld_path_url += "/"
-    bld_path_url += Path(os.getenv('CONDA_BLD_PATH')).as_posix()
-    return bld_path_url
-
-
-def _definitions():
-    condarc = _get_condarc()
-    definitions = {
-        "name": APP,
-        "company": "Spyder-IDE",
-        "reverse_domain_identifier": "org.spyder-ide.Spyder",
-        "version": str(SPYVER),
-        "environment_file": str(BASE_LOCK_FILE),
-        "installer_filename": OUTPUT_FILE.name,
-        "installer_type": args.install_type,
-        "initialize_by_default": False,
-        "initialize_conda": False,
-        "register_python": False,
-        "register_envs": False,
-        "extra_envs": {
-            "spyder-runtime": {
-                "environment_file": str(RT_LOCK_FILE),
-            }
-        },
-        "channels_remap": [
-            {
-                "src": _get_conda_bld_path_url(),
-                "dest": "https://conda.anaconda.org/conda-forge"
-            }
-        ]
-    }
-
-    definitions["license_file"] = str(RESOURCES / "bundle_license.rtf")
-    if args.install_type == "sh":
-        definitions["license_file"] = str(SPYREPO / "LICENSE.txt")
-
-    if LINUX:
-        definitions.update(
-            {
-                "default_prefix": os.path.join(
-                    "$HOME", ".local", INSTALLER_DEFAULT_PATH_STEM
-                ),
-                "pre_install": str(RESOURCES / "pre-install.sh"),
-                "post_install": str(RESOURCES / "post-install.sh"),
-                "extra_files": [
-                    {str(RESOURCES / "bundle_readme.md"): "README.txt"},
-                    {condarc: ".condarc"},
-                    {str(RESOURCES / "menuinst_cli.py"): "bin/menuinst_cli.py"},
-                ],
-            }
+        welcome_text = RESOURCES / "osx_pkg_welcome.rtf"
```

Comment:
Perhaps we should add a mention to the `uninstall-spyder` command to this doc? I can't think of a better place to let users know about it during the installation process.

Reply:
I like this suggestion. I'll add it.